selfdrive/car/interfaces.py

```diff
@@ -98,6 +98,12 @@ def get_params(cls, candidate: str, fingerprint: Optional[Dict[int, Dict[int, in
     ret = CarInterfaceBase.get_std_params(candidate)
     ret = cls._get_params(ret, candidate, fingerprint, car_fw, experimental_long)
 
+    # Set maximum allowed steering torque
+    # TODO: the interfaces should set carControlParams
+    CarControllerParams = get_interface_attr("CarControllerParams")[ret.carName]
+    if CarControllerParams is not None and ret.steerControlType != car.CarParams.SteerControlType.angle:
+      ret.carControlParams.steerMax = CarControllerParams(ret).STEER_MAX
```

Comment:
@adeebshihadeh are you fine with this cereal structure? Plan is to replace the classes with carParams eventually